src/main/resources/feedbackQuestionContribSubmissionFormTemplate.html

```diff
@@ -1,9 +1,9 @@
-<div class="col-sm-3 mobile-no-padding">
+<div class="col-sm-3 col-no-padding">
   <select class="form-control" name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}" ${disabled}>
     ${contribSelectFragmentsHtml}
   </select>
 </div>
-<div>
+<div class="col-sm-6">
```

Comment:
since u changed this part, probably (if u want to i.e. if u think this indent is a bit ugly, if not its fine) you will want to remove this indent of the more info link. 

![image](https://user-images.githubusercontent.com/28526038/40930271-c5177070-6859-11e8-8fa1-0da8df24dba2.png)

Reply:
forgot to add mobile no padding class. Will change that